app/models/concerns/twitter_user/api.rb

```diff
@@ -0,0 +1,262 @@
+    end
+
+    def inactive_friends
+      ExTwitter::Client.new._extract_inactive_users(friends, authorized: ego_surfing?)
+    end
+
+    def inactive_followers
+      ExTwitter::Client.new._extract_inactive_users(followers, authorized: ego_surfing?)
+    end
+
+    def clusters_belong_to
+      text = statuses.map{|s| s.text }.join(' ')
+      ExTwitter::Client.new.clusters_belong_to(text)
+    end
+
+    def close_friends(options = {})
+      min = options.has_key?(:min) ? options.delete(:min) : 1
+      client.close_friends(__uid_i, options.merge(
+        min: min,
+        replying: replying(options.merge(uniq: false)),
+        replied: replied(uniq: false),
+        favoriting: favoriting(options.merge(uniq: false)))
+      ).map { |u| u.uid = u.id; u }
+    end
+
+    def inactive_friends_graph
+      inactive_friends_size = inactive_friends.size
+      friends_size = friends_count
+      [
+        {name: I18n.t('legend.inactive_friends'), y: (inactive_friends_size.to_f / friends_size * 100)},
+        {name: I18n.t('legend.not_inactive_friends'), y: ((friends_size - inactive_friends_size).to_f / friends_size * 100)}
+      ]
+    end
+
+    def inactive_followers_graph
+      inactive_followers_size = inactive_followers.size
+      followers_size = followers_count
+      [
+        {name: I18n.t('legend.inactive_followers'), y: (inactive_followers_size.to_f / followers_size * 100)},
+        {name: I18n.t('legend.not_inactive_followers'), y: ((followers_size - inactive_followers_size).to_f / followers_size * 100)}
+      ]
+    end
+
+    def removing_graph
+      large_rate = [removing.size * 10, 100].min
+      [
+        {name: I18n.t('legend.large'), y: large_rate},
+        {name: I18n.t('legend.small'), y: 100 - large_rate}
+      ]
+    end
+
+    def removed_graph
+      large_rate = [removed.size * 10, 100].min
+      [
+        {name: I18n.t('legend.large'), y: large_rate},
+        {name: I18n.t('legend.small'), y: 100 - large_rate}
+      ]
+    end
+
+    def replying_graph
+      friends_size = friends_count
+      replying_size = replying.size
+      [
+        {name: I18n.t('legend.replying'), y: (replying_size.to_f / friends_size * 100)},
+        {name: I18n.t('legend.others'), y: ((friends_size - replying_size).to_f / friends_size * 100)}
+      ]
+    end
+
+    def replied_graph
+      followers_size = followers_count
+      replied_size = replied.size
+      [
+        {name: I18n.t('legend.replying'), y: (replied_size.to_f / followers_size * 100)},
+        {name: I18n.t('legend.others'), y: ((followers_size - replied_size).to_f / followers_size * 100)}
+      ]
+    end
+
+    def favoriting_graph
+      friends_size = friends_count
+      favoriting_size = favoriting.size
+      [
+        {name: I18n.t('legend.favoriting'), y: (favoriting_size.to_f / friends_size * 100)},
+        {name: I18n.t('legend.others'), y: ((friends_size - favoriting_size).to_f / friends_size * 100)}
```

Comment:
__[rubocop]__
* Space inside { missing. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)
* Line is too long. [104/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)
* Space inside } missing. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)